frontend/src/app/header/header.component.html

```diff
@@ -12,6 +12,11 @@
           Scenarios
         </a>
       </li>
+      <li class="nav-item" routerLinkActive="active">
+        <a class="nav-link" routerLink="/workflows">
+          Workflows
+        </a>
```

Comment:
```suggestion
          <span class="badge text-white bg-gradient-primary">New</span>
        </a>
```

Reply:
Had to think a bit what this was supposed to be. But yeah workflows got the colorful new badge now